Solutions/Authomize/Analytic Rules/Admin_password_wasnt_updated.yaml

```diff
@@ -0,0 +1,62 @@
+id: 63d87fcb-d197-48d2-a642-de4813f0219a
+name: Admin password wasn't updated during the last 30 days
+kind: Scheduled
+description: The policy detects admin account where password wasn't updated during the last 30 days.
```

Comment:
Can we add more information in the description. It is repeating the name itself here.

Reply:
Updated - I think is is pretty self explanatory.